addons/sale/tests/__init__.py

```diff
@@ -3,4 +3,3 @@
 import test_sale_to_invoice
 import test_sale_order
 from . import test_product_id_change
-import test_sale_ui
```

Comment:
Those are moved in sales_management. Actually sales module will contain the logic, sales_management will be the app enabling its use. The purpose is that when you use the ecommerce, you don't have Sales + Invoice + Website apps, only Invoice and Website. You may activate sales feature by installing the sales management app that basically enables the display of various menus. In short: nothing changes, the purpose is to have less apps displayed in some specific use (i.e. i use only ecommerce, I use only Sales management, ...) .